include/mbedtls/config_psa.h

```diff
@@ -102,6 +102,7 @@
 #define PSA_WANT_ALG_SHA3_256 1
 #define PSA_WANT_ALG_SHA3_384 1
 #define PSA_WANT_ALG_SHA3_512 1
+#endif
```

Comment:
Sure! However that commit belong to the preceding PR (#8033), so I will fix that one and rebase this one on top of it.

Since we are talking about it and you already reviewed this one, can you please take a look also to #8033? Commits there are the initial ones of this PR, so there won't be anything new...

Reply:
#8033 needs this squashing as well.

Reply:
Yes, correct, indeed I rebased/modified #8033 last friday, so this unnecessary commit is no more listed in its history